lib/commands/project.js

```diff
@@ -668,10 +673,17 @@ project.deploy = function (projectName, dir, includeModules, registry, withTests
       });
     },
     function deployProject (selectedProject, fn) {
-      if (!selectedProject) {
-        return fn('You must deploy to a project.');
-      }
-      projectController.deploy(selectedProject.id, dir, includeModules, registry, withTests, function (err, domain) {
+      if (!selectedProject) return fn('You must deploy to a project.');
+
+      options = {
+        registry: registry,
+        withTests: !!withTests,
```

Comment:
@tzmanics It's noted [here](https://github.com/onmodulus/modulus-cli/pull/139/files#diff-e5001988c284bf49302c6986b4b88a3bL312). I just moved the checking to a different file. Also it's to make it a boolean if it already isn't one.